draft-ietf-tls-tls13.md

```diff
@@ -3264,17 +3273,18 @@ by an encrypted body, which itself contains a type and optional padding.
        } TLSInnerPlaintext;
 
        struct {
-           ContentType opaque_type = 23; /* application_data */
-           ProtocolVersion legacy_record_version = 0x0301; /* TLS v1.x */
-           uint16 length;
+           uint16 masked_length;
```

Comment:
A 32-bit header would be better for crypto off-loading, but 16-bits is better than 5 bytes.